src/bwt_util.rs

```diff
@@ -95,4 +201,12 @@ mod tests {
         let bwt_stream = naive_bwt(&data);
         assert_eq!(bwt_stream, "AACC$A$");
     }
-}
\ No newline at end of file
+
+    #[test]
+    fn merging_paper_example_works() {
+        let bwt0 = "AC$CA".as_bytes();
+        let bwt1 = "AAAC$".as_bytes();
+        let merged_bwt = pairwise_bwt_merge(bwt0, bwt1);
+        assert_eq!(merged_bwt, "AACAAC$C$A".as_bytes())
+    }
```

Comment:
Good to see this test working.  We probably want some more complicated ones also with different length strengths, high similarity count, etc.  I think we could create a function like `naive_merge_bwt` that basically calls `naive_bwt` on each individual input and then merges that into the BWT (this is obviously not optimal, it's just for testing).

Then each complicated larger test can be of this form:
```
let dataset = ...; // hard code, generate, etc.
let naive_result = naive_bwt(&dataset);
let naive_merge_result = naive_merge_bwt(&dataset);
assert_eq!(naive_result, naive_merge_result);
```

Some example tests that hit some of the weird corners can be found starting here: https://github.com/HudsonAlpha/rust-msbwt/blob/077089ab2c30c2722e7530d08fdb02051efa78d0/src/dynamic_bwt.rs#L530

Reply:
Nope, that's great, just wanted to see some of those things that hit the corners.  Also, that extra function will make it much easier to add tests if we encounter anything weird in the future.